support.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>AdoptOpenJDK - GetOpenJDK - Support</title>
+  <link rel="shortcut icon" href="dist/assets/favicon.ico" type="image/x-icon">
+  <link rel="icon" href="dist/assets/favicon.ico" type="image/x-icon">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
+  <link rel="stylesheet" href="dist/css/styles.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css">
+</head>
+
+<body class="blue-bg">
+
+  <nav>
+    <a id="logo" href="./index.html" class="a-button">
+      <div><img src="dist/assets/squares-logo.png"></div>
+      <div><span>AdoptOpenJDK - GetOpenJDK</span></div>
+    </a>
+  </nav>
+
+  <!-- TODO Style should match installation page, it doesn't quite... -->
+  <main class="grey-bg">
+
+    <h1 class="margin-bottom">Support</h1>
+
+    <div id="support-container">
+      <div class="margin-bottom">
+        <h2 class="bold">WARNING!</h2>
+        <p id="notforproduction">
+          The AdoptOpenJDK OpenJDK binaries are <strong>not</strong> fully tested (i.e. JCK/TCK + further testing) and verified. If you want a foramlly tested
+          and verified Java binary then you can get one from vendors such as Oracle, Azul, IBM, RedHat, SAP and others.
```

Comment:
"RedHat" -> "Red Hat"

Like acceptance speeches, once you start listing people you are going to have to try and be fair, and comprehensive, or perhaps consider not mentioning anybody.  While it is reasonable to give caveats about this build, perhaps we don't express an opinion on others.